jujugui/views.py

```diff
@@ -98,6 +98,7 @@ def config(request):
         'viewContainer': '#main',
         'baseUrl': base_url,
         'transitions': False,
+        # XXX kadams54: cachedFonts doesn't appear to be used anywhere.
```

Comment:
I could say the same about line 125 :wink:. It didn't appear to be used anywhere based on a quick grep through the code, but I'd want to do a more thorough investigation before removing.